File: 2011. Final Value of Variable After Performing Operations/finalValueAfterOperations.js
/**
 * @param {string[]} operations
 * @return {number}
 */
var finalValueAfterOperations = function (operations) {
    let x = 0

    for (let oper of operations) {
        if (oper === "++X" || oper === "X++") {
            x++;
        }
        else {
            x--;
        }
    }

    return x;
};

File: 2011. Final Value of Variable After Performing Operations/finalValueAfterOperations.cpp
#include <iostream>
#include <vector>

using namespace std;

class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int x = 0;

        for(auto oper : operations){
            if(oper == "++X" || oper == "X++"){
                x++;
            }
            else{
                x--;
            }
        }

        return x;
    }
};